src/lib/websocket/index.ts

```diff
@@ -0,0 +1,293 @@
+import WebSocket, { Server, Data } from 'ws';
+import * as http from 'http';
+import { SkyraClient } from '../SkyraClient';
+import Collection from '@discordjs/collection';
+import { enumerable } from '../util/util';
+import { Util } from 'klasa-dashboard-hooks';
+import { CLIENT_SECRET } from '../../../config';
+import { KlasaUser } from 'klasa';
+import { Events } from '../types/Enums';
+
+
+// TODO - should we timeout connections? disconnect after X period?
+// TODO - after the connection of a DashboardWebsocketUser is closed, is that class instance GC'd? should be
```

Comment:
```suggestion
// TODO (Magnaboy): After the connection of a DashboardWebsocketUser is closed, is that class instance GC'd? should be
```

Conventional way of writing TODO messages